v0/destinations/facebook_pixel/transform.js

```diff
@@ -427,10 +515,21 @@ const processEvent = (message, destination) => {
       stats.increment("fb_pixel_timestamp_error", 1, {
         destinationId: destination.ID
       });
-      throw new CustomError(
-        "[facebook_pixel]: Events must be sent within seven days of their occurrence or up to one minute in the future.",
-        400
-      );
+      throw new ErrorBuilder()
+        .setMessage(
+          "[facebook_pixel]: Events must be sent within seven days of their occurrence or up to one minute in the future."
```

Comment:
The prefix is a little out of place and redundant.
```suggestion
          "Events must be sent within seven days of their occurrence or up to one minute in the future."
```